qlty-plugins/plugins/linters/golangci-lint/plugin.toml

```diff
@@ -32,6 +33,17 @@ output_format = "golangci_lint"
 suggested = "targets"
 output_missing = "parse"
 
+[[plugins.definitions.golangci-lint.drivers.lint.version]]
+version_matcher = ">=2.0.0"
```

Comment:
The version matcher above has `<=` and this one has `>=` so they overlap on `2.0.0`